Platform/MeteorlakeBoardPkg/CfgData/CfgData_Memory.yaml

```diff
@@ -42,56 +42,56 @@
       help         : >
                      Set Dqs mapping relationship between CPU and DRAM, Memory controller 0 Channel 0- board-dependentt
       length       : 0x02
-      value        : { 0, 1}
+      value        : { 0, 0}
   - DqsMapCpu2DramMc0Ch1 :
       name         : Dqs Map CPU to DRAM MC 0 CH 1
       type         : EditNum, HEX, (0x00,0xFFFF)
       help         : >
                      Set Dqs mapping relationship between CPU and DRAM, Memory controller 0 Channel 1- board-dependentt
       length       : 0x02
-      value        : { 0, 1}
+      value        : { 0, 0}
   - DqsMapCpu2DramMc0Ch2 :
       name         : Dqs Map CPU to DRAM MC 0 CH 2
       type         : EditNum, HEX, (0x00,0xFFFF)
       help         : >
                      Set Dqs mapping relationship between CPU and DRAM, Memory controller 0 Channel 2- board-dependentt
       length       : 0x02
-      value        : { 0, 1}
+      value        : { 0, 0}
   - DqsMapCpu2DramMc0Ch3 :
       name         : Dqs Map CPU to DRAM MC 0 CH 3
       type         : EditNum, HEX, (0x00,0xFFFF)
       help         : >
                      Set Dqs mapping relationship between CPU and DRAM, Memory controller 0 Channel 3- board-dependentt
       length       : 0x02
-      value        : { 0, 1}
+      value        : { 0, 0}
   - DqsMapCpu2DramMc1Ch0 :
       name         : Dqs Map CPU to DRAM MC 1 CH 0
       type         : EditNum, HEX, (0x00,0xFFFF)
       help         : >
                      Set Dqs mapping relationship between CPU and DRAM, Memory controller 1 Channel 0- board-dependentt
       length       : 0x02
-      value        : { 0, 1}
+      value        : { 0, 0}
   - DqsMapCpu2DramMc1Ch1 :
       name         : Dqs Map CPU to DRAM MC 1 CH 1
       type         : EditNum, HEX, (0x00,0xFFFF)
       help         : >
                      Set Dqs mapping relationship between CPU and DRAM, Memory controller 1 Channel 1- board-dependentt
       length       : 0x02
-      value        : { 0, 1}
+      value        : { 0, 0}
   - DqsMapCpu2DramMc1Ch2 :
       name         : Dqs Map CPU to DRAM MC 1 CH 2
       type         : EditNum, HEX, (0x00,0xFFFF)
       help         : >
                      Set Dqs mapping relationship between CPU and DRAM, Memory controller 1 Channel 2- board-dependentt
       length       : 0x02
-      value        : { 0, 1}
+      value        : { 0, 0}
```

Comment:
should update them in dlt file not to impact other platforms.

Reply:
fixed, Updated value in mtlp dlt file instead